test/cpp/jit/test_gpu.cpp

```diff
@@ -3066,80 +3066,188 @@ void testGPU_FusionSimpleGemm() {
   }
 }
 
-// This test currently requires a combination of broadcast and reduction
-// operations and parellelization strategy that is currently not supported.
-// It is a goal to get this example working and this test is added so we
-// can continue working on getting this example fixed. Right now it
-// produces an incorrect result. Either we need to error coherently on the
-// optimization strategy we don't support and set this test to one we do support
-// or we need to get this schedule working correctly.
+// Softmax without input normalization. This is a simplified version
+// of testGPU_FusionSoftmaxNormalized.
 void testGPU_FusionSoftmax() {
   torch::jit::fuser::cuda::CudaKernel prog;
   Fusion& fusion = *prog.fusion_;
   FusionGuard fg(&fusion);
 
+  const int tidx = 32;
+  const int dimx = 32;
+  const int dimy = 16;
+  const int dimz = 130;
+
   // Set up your input tensor views
   TensorView* input_tv0 = makeDummyTensor(3);
   fusion.addInput(input_tv0);
 
+  TensorView* exp_tv4 = unaryOp(UnaryOpType::Exp, input_tv0);
+  TensorView* sum_exp_tv5 =
+      reductionOp(BinaryOpType::Add, {2}, new Float(0), exp_tv4);
+  TensorView* bcast_sum_tv6 = broadcast(sum_exp_tv5, {false, false, true});
```

Comment:
would `broadcast(sum_exp_tv5, {true})` work?

Reply:
I don't think so. The size of the second argument list needs to be the dimensionality of the output tensor, so I believe the only valid value is `{false, false, true}`.